app/theme/brave_theme_resources.grd

```diff
@@ -110,6 +110,10 @@
         <structure type="chrome_scaled_image" name="IDR_BRAVE_SEARCH_CONVERSION_BANNER_GRAPHIC" file="brave/brave_search_conversion_banner_graphic.png" />
         <structure type="chrome_scaled_image" name="IDR_BRAVE_SEARCH_CONVERSION_BANNER_GRAPHIC_DARK" file="brave/brave_search_conversion_banner_graphic_dark.png" />
         <structure type="chrome_scaled_image" name="IDR_BRAVE_SEARCH_LOGO_IN_SEARCH_PROMOTION" file="brave/brave_search_logo_in_search_promotion.png" />
+
+        <structure type="chrome_scaled_image" name="IDR_TAB_DROP_LEFT" file="tab_drop_left.png" />
+        <structure type="chrome_scaled_image" name="IDR_TAB_DROP_RIGHT" file="tab_drop_right.png" />
+
```

Comment:
Instead of adding new resources, transform the upstream image resource programmatically would be better, so that we don't have to follow up when upstream changes the resource.

I think this one can help you with that
https://source.chromium.org/chromium/chromium/src/+/main:ui/gfx/image/image_skia_operations.h;l=104;drc=ca5848749bd272becb90e826d701f4c6ab39bf13

Reply:
I think this still needs to be addressed. I thought we can avoid adding new resources. 

Can you use this resource and transform the direction on the runtime?
https://source.chromium.org/chromium/chromium/src/+/main:chrome/app/theme/theme_resources.grd;l=312?q=IDR_TAB_DROP_DOWN%20f:grd&ss=chromium%2Fchromium%2Fsrc

Reply:
Oops, I seem to be looking at the old commit. NVM ☺️